charts/webhook-dispatcher/templates/webhook-dispatcher-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-dispatcher-svc
+  labels:
+    app: webhook-dispatcher
+spec:
+  type: ClusterIP
+  ports:
+  - port: 80
```

Comment:
ingress does tls-termination, right? so we have to listen on port 80, but the actual endpoint is accesible through port 443 / with TLS?

Reply:
443 is terminated at ingress; forwards it to the service on port 80, which forwards it to 8080